<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="19720" started-at="2020-03-15T05:05:31Z" finished-at="2020-03-15T05:05:51Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="19720" started-at="2020-03-15T05:05:31Z" finished-at="2020-03-15T05:05:51Z">
      <class name="testcases.Logintest">
        <test-method status="PASS" signature="SetUp()[pri:0, instance:testcases.Logintest@55f3ddb1]" name="SetUp" is-config="true" duration-ms="5832" started-at="2020-03-15T05:05:31Z" finished-at="2020-03-15T05:05:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="PASS" signature="test01()[pri:0, instance:testcases.Logintest@55f3ddb1]" name="test01" duration-ms="508" started-at="2020-03-15T05:05:37Z" finished-at="2020-03-15T05:05:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test01 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.Logintest@55f3ddb1]" name="tearDown" is-config="true" duration-ms="2325" started-at="2020-03-15T05:05:37Z" finished-at="2020-03-15T05:05:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.Logintest -->
      <class name="testcases.NewTest">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testcases.NewTest@8bd1b6a]" name="beforeMethod" is-config="true" duration-ms="4442" started-at="2020-03-15T05:05:39Z" finished-at="2020-03-15T05:05:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="f()[pri:0, instance:testcases.NewTest@8bd1b6a]" name="f" duration-ms="1" started-at="2020-03-15T05:05:44Z" finished-at="2020-03-15T05:05:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testcases.NewTest@8bd1b6a]" name="afterMethod" is-config="true" duration-ms="106" started-at="2020-03-15T05:05:44Z" finished-at="2020-03-15T05:05:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testcases.NewTest@8bd1b6a]" name="beforeMethod" is-config="true" duration-ms="4378" started-at="2020-03-15T05:05:44Z" finished-at="2020-03-15T05:05:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="test02()[pri:0, instance:testcases.NewTest@8bd1b6a]" name="test02" duration-ms="11" started-at="2020-03-15T05:05:48Z" finished-at="2020-03-15T05:05:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [JavaByKiran | Dashboard] but found [JavaByKiran | Log in]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [JavaByKiran | Dashboard] but found [JavaByKiran | Log in]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at testcases.NewTest.test02(NewTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test02 -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testcases.NewTest@8bd1b6a]" name="afterMethod" is-config="true" duration-ms="2066" started-at="2020-03-15T05:05:48Z" finished-at="2020-03-15T05:05:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testcases.NewTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
